src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,33 @@
+package lotto
+
+class LottoMachine(private val lottoNumberGenerator: LottoNumberGenerator) {
+    fun generateLotto(amount: Amount): List<Lotto> {
+        val lottoCount = amount.divide(LOTTO_UNIT_PRICE)
+        val lottos = mutableListOf<Lotto>()
+        repeat(lottoCount) {
+            val lottoNumbers = generateLottoNumbers()
+            lottos.add(Lotto(lottoNumbers))
+        }
+        return lottos
```

Comment:
```suggestion
        return (0 until lottoCount).map { Lotto(generateLottoNumbers()) }
```
와 같은 형태로도 만들 수 있습니다 :)

Reply:
오 저 부분 되게 신기하네요
감사합니다! 🙇